tests/test_single_topology.py

```diff
@@ -2005,3 +2005,42 @@ def test_chiral_bond_breaking_1_core_1_dummy(seed):
 
     # should not raise an assertion
     verify_chiral_validity_of_core(mol_a, mol_b, core, ff)
+
```

Comment:
Not in diff, but I can't tell if `test_canonicalize_improper_idxs` in this file is correct : looks like it still assumes `(center, p0, p1, p2)` ?

https://github.com/proteneer/timemachine/blob/be34fbf458b54de5454564b67aeb1d70450a1b3f/tests/test_single_topology.py#L524-L537

Reply:
Thanks - good catch, resolved by 5a75089959f4b8348734c3d6d01a98c887609762